src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -235,6 +235,13 @@ public string HubName
         /// </summary>
         public AppLeaseOptions AppLeaseOptions { get; set; } = AppLeaseOptions.DefaultOptions;
 
+        /// <summary>
+        /// Option to control the receive message size in bytes of the grpc client, which is used by Durable Funsiont c# Isolated and Java.
+        /// Defaults to 4,194,304 (4 MB).
+        /// The maximum allowable value is <see cref="int.MaxValue"/>, which corresponds to the durable grpc server's receive limit.
```

Comment:
our grpcmessage receive limit at [server](https://github.com/Azure/azure-functions-durable-extension/blob/e211fbbe3a28cb61932f386dee2fbbd98cc05439/src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs#L64) is int.maxvalue, so same as client